### PR TITLE
Updates to package.json: rev version, add WKT, and recateogrize dependencies

### PR DESCRIPTION
**Pull request by haberman**

### CODE REVIEW

**Comment by haberman** [comment]
This failure here is spurious. @joeltine want to take a look and give LGTM if it LGTM? :)

File: js/package.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  "version": "3.0.0-alpha.6",
+  "version": "3.0.0-alpha.6.2",
   "description": "Protocol Buffers for JavaScript",
   "main": "google-protobuf.js",
-  "dependencies": {
+  "files": "google",
+  "dependencies": {},
+  "devDependencies": {
+    "glob": "~6.0.4",
+    "google-closure-compiler": "~20151216.2.0",
```
Comment (joeltine) [inline js/package.json]: This is a very old compiler dependency.

File: js/package.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
 {
   "name": "google-protobuf",
-  "version": "3.0.0-alpha.6",
+  "version": "3.0.0-alpha.6.2",
   "description": "Protocol Buffers for JavaScript",
   "main": "google-protobuf.js",
-  "dependencies": {
+  "files": "google",
```
Comment (joeltine) [inline js/package.json]: "files" is supposed to be an array? https://docs.npmjs.com/files/package.json#files

**Comment by joeltine** [comment]
FYI: maybe run through http://package-json-validator.com/

**Comment by joeltine** [comment]
Minus the comments, LGTM. All are optional... unless the `files` issue actually breaks npm publish.

File: js/package.json (hunk 1 above)
Comment (haberman) [inline js/package.json]: Done.

File: js/package.json (hunk 2 above)
Comment (haberman) [inline js/package.json]: Done.

**Comment by haberman** [comment]
Good suggestions, thanks!

**Pull request merged**